README.md

```diff
@@ -0,0 +1,35 @@
+README:
+
+Team mike's UML Editor
+
+Team mike's UML Editor is an application that allows the user to add, delete, and edit UML
+classes. Each UML class can hold different fields and methods, which can be added, deleted,
+and renamed. The user can also save files and load other ones to easily save progress and 
+continue to work on it at a later time. The application currently runs only using a Command
+Line Interface.
+
+Requirements to use the application:
+
+Java 11
+Gradle
+
+How to run the application:
+
+1. Navigate to the directory with the files and have a command prompt ready
+2. Type these commands in the command prompt:
+gradle build
+gradle shadowJar
+java -jar ./build/libs/mike-all.jar
```

Comment:
Did a quick search on editing and formatting readmes in github, and we can make this look like a bulleted list if we do:

2. Type these commands in the command prompt:
    - gradle build
    - gradle shadowJar
    - java -jar ./build/libs/mike-all.jar

The spacing of the dashes is important, apparently. They need to be "underneath the first character of the previous thing in the list" according to [the github site on basic writing and formatting] (https://docs.github.com/en/enterprise/2.20/user/github/writing-on-github/basic-writing-and-formatting-syntax) at about half-way down the page.